jsmith/artwork.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <link href="css/style.css" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css?family=Bungee+Inline" rel="stylesheet">
+  <title> Artwork </title>
+</head>
+
+<body>
+  <div class="nav">
+    <ul>
+      <li class="nav-link"><a href="index.html">Home</a></li>
+      <li class="nav-link"><a href="artwork.html">Artwork</a></li>
+      <li class="nav-link"><a href="http://saskartists.ca/">Sask Artists</a></li>
+    </ul>
+  </div>
+
+  <div class="iframe-container">
+    <iframe src="https://docs.google.com/presentation/d/1s19STZd_oo30oqsBFke_GGOBmtMC4DfRK0mp-st69oI/embed?start=false&loop=false&delayms=10000" frameborder="0" width="960" height="569"></iframe>
```

Comment:
So all of the artwork is inside of a Google Slides presentation, which we view through an iframe? If so, do we need to keep a copy of the images on our local folder? This is a moderately important issue, as server storage is a bit of a concern, but completeness is as well, particularly if something should change significantly with Google Slides. 
As well, it is impossible for anyone except for you to access the slideshow, making it very difficult to maintain in the future. You may want to consider simply going with a less fancy route and simply put the images in.